pytorch_lightning/plugins/native_amp.py

```diff
@@ -52,7 +53,10 @@ def backward(self, closure_loss, optimizer, opt_idx, *args, **kwargs):
 
         # unscale gradient to allow analyze within `on_after_backward`
         if not self.trainer.train_loop.should_accumulate() and automatic_optimization:
-            self.trainer.scaler.unscale_(optimizer)
+            if isinstance(optimizer, LightningOptimizer):
+                self.trainer.scaler.unscale_(optimizer._optimizer)
```

Comment:
shall we rather have define in PL opt `def __call__() -> optimizer._optimizer`?

Reply:
I move the _unscale to optimizer_step. Let's see if it doesn't break the test.